bin/load_exp_design.sh

```diff
@@ -3,6 +3,9 @@ set -e
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}" )" &> /dev/null && pwd )
 source "${SCRIPT_DIR}/common_routines.sh"
 
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]:-$0}" )" && pwd )
```

Comment:
Is this replacing line 3-4?
If yes, then shall we remove those line, please?
Thanks

Reply:
Thanks, done in a71412ddbbad1d7721d658a277ee9257e3efb576